cron-backup/grafana_backup.sh

```diff
@@ -4,55 +4,55 @@
+        echo "STATUS: Grafana backup is Failed." >> /tmp/grafana.txt
+        echo " " >> /tmp/grafana.txt
+	echo "The source backup directory: grafana_src is not available" >> /tmp/grafana.txt
+        < /tmp/grafana.txt mail -s "${SOURCE_NAME}: Grafana Data Backup" "${CRON_BACKUP_MAIL}"
 	exit
 else
-	tar cvzf /var/lib/backup/"${SOURCE_NAME}"_grafana_data_backup_"${DATE1}".tgz ${grafana_src}/
+	tar cvzf /var/lib/backup/grafana/"${SOURCE_NAME}"_grafana_data_backup_"${DATE1}".tgz ${grafana_src}/
 fi
 
-s3cmd sync --no-mime-magic /var/lib/backup/*grafana*.tgz s3://"${S3_BUCKET_GRAFANA}"/
+s3cmd put -r --no-mime-magic /var/lib/backup/grafana/ s3://"${S3_BUCKET_GRAFANA}"/
 
 # Moving the backup to S3 bucket
 if [ $? -eq 0 ]; then
 
-        echo "DATE:" $DATE > /grafana.txt
-        echo " " >> /grafana.txt
-        echo "DESCRIPTION: ${SOURCE_NAME}_Grafana backup" >> /grafana.txt
-        echo " " >> /grafana.txt
-        echo "STATUS: Grafana backup is Successful." >> /grafana.txt
-        echo " " >> /grafana.txt
-        echo "******* Grafana Data Backup ****************" >> /grafana.txt
-        echo " " >> /grafana.txt
-        s3cmd ls --no-mime-magic s3://"${S3_BUCKET_GRAFANA}"/  --human-readable | grep -i "${SOURCE_NAME}"_grafana_data | cut -d' ' -f3- | tac | head -10 | sed "s,s3:\/\/${S3_BUCKET_GRAFANA}\/,,g" &>> /grafana.txt
-        echo " " >> /grafana.txt
-        echo "************** END **************************" >> /grafana.txt
+        echo "DATE:" $DATE > /tmp/grafana.txt
+        echo " " >> /tmp/grafana.txt
+        echo "DESCRIPTION: ${SOURCE_NAME}_Grafana backup" >> /tmp/grafana.txt
+        echo " " >> /tmp/grafana.txt
+        echo "STATUS: Grafana backup is Successful." >> /tmp/grafana.txt
+        echo " " >> /tmp/grafana.txt
+        echo "******* Grafana Data Backup ****************" >> /tmp/grafana.txt
+        echo " " >> /tmp/grafana.txt
+        s3cmd ls --no-mime-magic s3://"${S3_BUCKET_GRAFANA}"/  --human-readable | grep -i "${SOURCE_NAME}"_grafana_data | cut -d' ' -f3- | tac | head -10 | sed "s,s3:\/\/${S3_BUCKET_GRAFANA}\/,,g" &>> /tmp/grafana.txt
+        echo " " >> /tmp/grafana.txt
+        echo "************** END **************************" >> /tmp/grafana.txt
 
 else
-        echo "DATE:" $DATE > /grafana.txt
-        echo " " >> /grafana.txt
-        echo "DESCRIPTION: ${SOURCE_NAME}_Grafana backup" >> /grafana.txt
-        echo " " >> /grafana.txt
-        echo "STATUS: Grafana backup is Failed." >> /grafana.txt
-        echo " " >> /grafana.txt
-        echo "Something went wrong, Please check it"  >> /grafana.txt
-        < /grafana.txt mail -s "${SOURCE_NAME}: Grafana Data Backup" "${CRON_BACKUP_MAIL}"
+        echo "DATE:" $DATE > /tmp/grafana.txt
+        echo " " >> /tmp/grafana.txt
+        echo "DESCRIPTION: ${SOURCE_NAME}_Grafana backup" >> /tmp/grafana.txt
+        echo " " >> /tmp/grafana.txt
+        echo "STATUS: Grafana backup is Failed." >> /tmp/grafana.txt
```

Comment:
grammar: "Grafana backup failed." is the proper wording here.

Reply:
It's done.